qiskit/visualization/state_visualization.py

```diff
@@ -186,17 +186,20 @@ def plot_state_hinton(state, title='', figsize=None, ax_real=None, ax_imag=None,
         return fig
 
 
-def plot_bloch_vector(bloch, title="", ax=None, figsize=None):
+def plot_bloch_vector(bloch, title="", ax=None, figsize=None, *, coord_type="cartesian"):
```

Comment:
why is the `*` needed?

Reply:
Apologies, I was trying out different ways to accept the argument. The * can be removed.

Reply:
Fixed it in a recent commit